app/integration_action.go

```diff
@@ -325,27 +325,34 @@ func (w *LocalResponseWriter) WriteHeader(statusCode int) {
 	w.status = statusCode
 }
 
-func (a *App) DoLocalRequest(rawURL string, body []byte) (*http.Response, *model.AppError) {
+func (a *App) doPluginRequest(method, rawURL string, values url.Values, body []byte) (*http.Response, *model.AppError) {
```

Comment:
1/5 maybe as a separate ticket - I _think_ we have several places in the server code where plugin requests are composed; locate and consolidate?

Reply:
I'll create a ticket.